### ISSUE TITLE
Windows last building step fails with no output

### ISSUE THREAD

**Issue by Kavatch** [Eloston/ungoogled-chromium#546]
*If you have a problem, please fill out ALL the following fields.*

*Otherwise, please replace this template with your description.*

### Problem description
When executing the last command needed for building the browser it will end without any output.
So when I run 'py ungoogled_packaging\package.py' it ends directly without an output. When I run in in the python interface itself it just said:
`======= RESTART: G:\Chromium\build\src\ungoogled_packaging\package.py =======`

**Comment by Eloston** [comment]
Were you able to build Chromium successfully?

**Comment by Kavatch** [comment]
No, I got no zip file.

**Comment by Eloston** [comment]
Let me rephrase: Did `build.py` complete successfully? What did it say near the end of its output?

**Comment by Eloston** [comment]
Interesting, I didn't know Windows support was fixed. Are you able to run the browser from the build outputs? (e.g. located in `build/src/out/Default/chrome.exe`)

**Comment by Kavatch** [comment]
> Interesting, I didn't know Windows support was fixed. Are you able to run the browser from the build outputs? (e.g. located in `build/src/out/Default/chrome.exe`)

It is not fixed, like I said it gave me an error but I just bypassed with some comments in one file to disable a feature which should not have been needed. (At least I hope it was not needed. I think it was "Safe Browsing Basic" ).
And there is sadly no chrome.exe file in my folder. 
But as I said, because I have bypassed the whole thing in such a way can of course also have led to the error. But what really surprised me was that it didn't even execute a single line from 'package.py'. I debugged it and it exited immediately after starting and didn't even execute the import.

**Comment by Eloston** [comment]
> And there is sadly no chrome.exe file in my folder.

Then the build has failed. Packaging won't do anything for you.

> But what really surprised me was that it didn't even execute a single line from 'package.py'. I debugged it and it exited immediately after starting and didn't even execute the import.

I have no idea why this happens if `build.py` is able to run.

I'm closing this for now, since it doesn't make sense to work on this until #475 is done. We can debug this issue if it occurs then.
